lib/commons/color/get-background-color.js

```diff
@@ -351,7 +352,8 @@ color.getBackgroundColor = function(elm, bgElms = [], noScroll = false) {
 	if (noScroll !== true) {
 		// Avoid scrolling overflow:hidden containers, by only aligning to top
 		// when not doing so would move the center point above the viewport top.
-		const alignToTop = elm.clientHeight - 2 >= window.innerHeight * 2;
+		const clientHeight = elm.getBoundingClientRect().height;
+		const alignToTop = clientHeight - 2 >= window.innerHeight * 2;
```

Comment:
`clientHeight` is 0 on inline elements with multiple rects. getBoundingClientRect does give the rect for the entire thing.